container_pipeline/pipeline.py

```diff
@@ -75,6 +50,11 @@ def create_project(appid, jobid, repo_url, repo_branch, repo_build_path,
      target_file, notify_email, desired_tag, depends_on,
      test_tag) = sys.argv[1:]
     try:
-        create_project(*sys.argv[1:])
+        trigger_dockerfile_linter(appid, jobid, repo_url, repo_branch,
+                                  repo_build_path, target_file, notify_email,
+                                  desired_tag, depends_on, test_tag)
+        create_project(appid, jobid, repo_url, repo_branch, repo_build_path,
+                       target_file, notify_email, desired_tag, depends_on,
+                       test_tag)
     except OpenshiftError:
```

Comment:
I'm not sure if I understand what you mean here. But it seems you're pointing to the fact that this will create a project even if Dockerfile lint failed. 

To ensure that a project gets created in OpenShift only if linting succeeds, we should put this in `handle_lint_success` function of `workers/linter.py`. Because `trigger_dockerfile_lint` puts data onto the tube and passes the baton to lint worker. So, the part that triggers linting doesn't have any clue if lint failed or succeeded.